libgalois/include/katana/ArrowRandomAccessBuilder.h

```diff
@@ -162,6 +167,8 @@ class ArrowRandomAccessBuilder {
     builder_.SetValue(index, value);
   }
 
+  void SetNull(size_t index) { builder_.SetNull(index); }
```

Comment:
This looks good but I think a name that explicitly indicates the intention is better. This is not required while building a fresh array. This is only required when we're deleting a value right? Then why not call this `Delete`?

Reply:
What about `ResetNull`?

Reply:
I would propose `UnsetValue` since the initial state is null, but I don't feel strongly

Reply:
`UnsetValue` wins.

Reply:
I'd prefer `UnsetValue` or `DeleteValue`. 

`Reset` means reset to identity element (for reduction) in Gluon. It's better not to use "reset".